sample-apps/delivery-customizations-sample-app/web/frontend/pages/ExitIframe.jsx

```diff
@@ -0,0 +1,27 @@
+import { Redirect } from "@shopify/app-bridge/actions";
```

Comment:
Since all the files in `pages` are [inserted into the routes](https://github.com/Shopify/function-examples/pull/120/files#diff-c551d20308a7dd03554b86e0f27f135e1621421ce361199673255e37ffda46e9R34), we don't want to add components like this here. Doing this will add a route like `/ExitIframe`.

Instead, these components (i.e. `ExitIframe`, `NotFound`) should be moved to the `components` directory outside of `pages` (where `CustomizationForm` is).

Reply:
Hmm tried moving these, but I think vite is [using these for routes](https://github.dev/Shopify/function-examples/pull/120/files).